pallets/thea-executor/src/lib.rs

```diff
@@ -184,17 +189,17 @@ pub mod pallet {
 		/// Asset Metadata set ( config )
 		AssetMetadataSet(AssetMetadata),
 		/// Deposit Approved event ( Network, recipient, asset_id, amount, id))
-		DepositApproved(Network, T::AccountId, u128, u128, Vec<u8>),
+		DepositApproved(Network, T::AccountId, AssetId, u128, Vec<u8>),
 		/// Deposit claimed event ( recipient, asset id, amount, id )
-		DepositClaimed(T::AccountId, u128, u128, Vec<u8>),
+		DepositClaimed(T::AccountId, AssetId, u128, Vec<u8>),
 		/// Deposit failed event ( network, encoded deposit)
 		DepositFailed(Network, Vec<u8>),
 		/// Withdrawal Queued ( network, from, beneficiary, assetId, amount, id )
-		WithdrawalQueued(Network, T::AccountId, Vec<u8>, u128, u128, Vec<u8>),
+		WithdrawalQueued(Network, T::AccountId, Vec<u8>, AssetId, u128, Vec<u8>),
 		/// Withdrawal Ready (Network id )
 		WithdrawalReady(Network),
```

Comment:
is ready event still used? if so also output the vec<id> which are ready

Reply:
Backend doesn't care about it, so it's up to you to decide, if you want the ready event. It is now emitted if a withdrawal fails due to fee or other system level reason and user can try again.